core/src/main/java/com/adobe/aio/util/feign/FeignUtil.java

```diff
@@ -39,10 +39,11 @@ private FeignUtil() {
    */
   public static Feign.Builder getBaseBuilder() {
     return Feign.builder()
+        .logger(new Slf4jLogger())
+        //.logLevel(Level.BASIC)
```

Comment:
Should we pass a logger in here instead? As this is a util class, it may make sense to have a logger instance from the service using this FeignUtil class so that we can see from where the code was called, wdyt?

Reply:
Also could you remove unused fields from this class?

Reply:
as the logger yes I know it's sub-optimal, I created https://github.com/adobe/aio-lib-java/issues/146 for this.

Reply:
Thanks!